TLM/TLM/State/OptionsTabs/PoliciesTab_OnRoadsGroup.cs

```diff
@@ -0,0 +1,87 @@
+namespace TrafficManager.State {
+    using System;
+    using ColossalFramework.UI;
+    using ICities;
+    using TrafficManager.API.Traffic.Enums;
+    using TrafficManager.Manager.Impl;
+    using TrafficManager.UI;
+    using TrafficManager.UI.Helpers;
+    using TrafficManager.Util;
+
+    public static class PoliciesTab_OnRoadsGroup {
+
+        public static CheckboxOption BanRegularTrafficOnBusLanes =
+            new (nameof(Options.banRegularTrafficOnBusLanes), Options.PersistTo.Savegame) {
+                Label = "VR.Checkbox:Ban private cars and trucks on bus lanes",
+                Handler = OnBanRegularTrafficOnBusLanesChanged,
+            };
+
+        // at a segment to segment transition, only the smaller segment gets crossings
+        public static CheckboxOption NoDoubleCrossings =
+            new (nameof(Options.NoDoubleCrossings), Options.PersistTo.Savegame) {
+                Label = "VR.Option:No double crossings",
+                Handler = JunctionRestrictionsUpdateHandler,
+            };
+
+        private static UIDropDown _vehicleRestrictionsAggressionDropDown;
+
+        static PoliciesTab_OnRoadsGroup() {
+            try {
```

Comment:
Why this can fail? It should never fail in my opinion.

Reply:
Krzychu made same comment recently. The reason I do it is because if anything goes wrong in a static constructor it causes cascading failure in C:SL which does not guard against that eventuality. That in turn results in user's game config file being wiped, all mods are then disabled, paradox account reset, etc.

Errors are most likely to creep in during dev and test phase - while they will obviously get fixed it's a complete PITA to have to re-enable 140 mods, espeically when some mods (Ambient Sounds Tuner in particular) need enabling last to prevent further settings loss. So the try..catch is really just to protect myself from my own stupid bugs during dev & test :)